### DISCUSSION TITLE
[SC]: R006 errore interno

### DISCUSSION THREAD

**Discussion by Hdemografici** [italia/ansc#856] [Q&A] [answered]
### Ambiente

pre-produzione

### Servizio

R006 - Servizi cooperativi firma dichiarante

### idOperazione

n/a

### idEvento o idAnsc

8360266

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

colli al metauro

### Data del test (yyyy-mm-dd)

2024-05-03

### Descrivi il problema

errore interno senza ritorno di idOperazione, 
idOperazioneComune: SCH20240503151424217

https://**anscserviceval**.anpr.interno.it/services/service/firma/firmadichiarante/upload/1

ho controllato il payload sembra corretto, non riesco a capire cosa c'è che non va

### Come riprodurre il problema

n/a

### Payload (json)

```JSON
{
"testataRichiesta": { 
"idComune": "22758",
"idOperazioneComune": "SCH20240503151424217",
"dataOraRichiesta": "2024-05-03T15:14:24",
"nomeApplicativo": "Stato civile",
"versioneApplicativo": "08.37.05",
"fornitoreApplicativo": "HALLEY INFORMATICA srl"
},
"firmaDichiaranteInput": { 
"contenuto": "....",
"nomefile": "firma_Dichiarante.pdf",
"idEvento": "8360266",
"tipoFile": "1",
"tipoAllegato": "999",
"stato": "1",
"listaFirmatari": [ 
{ 
"cognome": "DE MARTINI",
"nome": "MANUEL",
"sesso": "M",
"dataNascita": "1980-07-06"
}
]
}
}
```

**Comment by mttfranci** [comment, marked as answer]
Salve,

Siete sicuri di richiamare questa url?
https://anscserviceval.anpr.interno.it/services/service/firma/firmadichiarante/upload/1

L'ambiente https://anscserviceval.* non è più supportato da tempo.

L'ambiente di preproduzione inizia invece con  : 

https://anscservicepre.*

Saluti

Assistenza ANSC

**Comment by Hdemografici** [comment]
effettivamente avevo qualche dubbio, mi sposto in ambiente pre-produzione e riprovo i vari servizi, grazie
